packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -181,7 +181,9 @@ export class MockRuntime
 			this.processBlobsP.reject(new Error("fake error"));
 		}
 		this.processBlobsP = new Deferred<void>();
-		await Promise.allSettled(blobPs).catch(() => {});
+		await Promise.allSettled(blobPs)
```

Comment:
Adding "then" here ensures processBlobs also runs onUploadResolve()

Reply:
this doesn't make much sense to me, is this behavior publicly described somewhere?

Reply:
I've seen different ordering of operations with these two approaches. Without the then() statement I've seen some tests failing when calling processAll() because the blobAttach op has not been sent by the time it calls processHandles() which means onUploadResolve has not been run by then. Adding then() was making this flakinness go away (it seems to me) and the ordering of some breakpoints I put in the code was always correct. I'll remove it for now and monitor any flakiness in the tests.